src/Views/AppListUpdateView.vala

```diff
@@ -95,11 +125,52 @@ namespace AppCenter.Views {
                 });
             });
 
+            client.notify["updates-number"].connect (() => {
+                if (client.updates_number > 0) {
+                    header_revealer.reveal_child = true;
+
+                    uint nag_numbers = 0U;
+                    uint64 update_real_size = 0ULL;
+                    bool using_flatpak = false;
+                    foreach (var package in get_packages ()) {
+                        if (package.update_available || package.is_updating) {
+                            if (package.should_pay) {
+                                nag_numbers++;
+                            }
+
+                            if (!using_flatpak && package.is_flatpak) {
+                                using_flatpak = true;
+                            }
+
+                            update_real_size += package.change_information.size;
+                        }
+                    }
+
+                    size_label.update (update_real_size, using_flatpak);
```

Comment:
I think you also need to update the size label in the constructor else it may not show if the view is opened after checking for update has finished.